src/transformer/events/mod_quiz/attempt_viewed.php

```diff
@@ -26,15 +26,17 @@ function attempt_viewed(array $config, \stdClass $event) {
     $course = $repo->read_record_by_id('course', $event->courseid);
     $lang = utils\get_course_lang($course);
 
+    $object = utils\get_activity\quiz_attempt($config, $event->id, $event->contextinstanceid);
+
+    // JISC specific activity type.
+    if (utils\is_enabled_config($config, 'send_jisc_data')) {
+        $object['definition']['type'] = 'http://xapi.jisc.ac.uk/activities/quiz';
```

Comment:
Hi @lzabo, Is this http://id.tincanapi.com/verb/viewed set if this is not enabled or is this a replacement?

Reply:
this is a replacement.
The request was: 
`Change activity type to: http://xapi.jisc.ac.uk/activities/quiz`

Reply:
I changed all the object to course_quiz.